subsys/bluetooth/host/smp.c

```diff
@@ -4158,7 +4158,7 @@ static uint8_t display_passkey(struct bt_smp *smp)
 }
 
 #if defined(CONFIG_BT_PERIPHERAL)
-static uint8_t smp_public_key_slave(struct bt_smp *smp)
+static uint8_t smp_public_key_periph(struct bt_smp *smp)
```

Comment:
See my other comment regarding shortening "peripheral" as well. 

```suggestion
static uint8_t smp_public_key_peripheral(struct bt_smp *smp)
```

Reply:
See my comment there as well